Updated Readme.md (#1)
ivailo1st/widgets: README.md

-   **Name** The displayname to show inside Colibo for widget-editors
-   **Description** A short description for the editor to choose the right widget.

> -   **Categories** A list of texts used for defining the different categories a widget is apart of

Reviewer: Maybe give a few examples of categories

Author: Added examples of categories.

> -   **PosterImage** A url used for linking to an image that is displayed as a poster of the widget in the widget library

Reviewer: ```suggestion
-   **PosterImage** A url to an image that is displayed as a poster of the widget in the widget library
```

Author: Done

> -   **ExampleImages** A list of urls that are used for linking images that are displayed as examples of how a widget looks like when its being used

Reviewer: ```suggestion
-   **ExampleImages** A list of urls to images that are displayed as examples of how a widget looks like when its being used
```

Author: Done
